### DISCUSSION TITLE
Welcome to PythonCFA Discussions!

### DISCUSSION THREAD

**Discussion by chanjuping** [PenangScienceCluster/PythonCFA#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chanjuping** [comment]
Let me get things rolling. I am a solutions architect who got started on my tech path being traumatised by the maintenance work required to keep a Windows ME machine running for 8 years with no backup, to getting my first laptop and installing Ubuntu on it, and along the way I got a degree in Forestry and a Masters in Biology. I enjoy learning things that catch my fancy, even if it is from scratch. I hope that perspective is able to properly serve you well on your Python and/or coding journey, through the course material in this repository.

If you have additional questions, or you have commentary that will help, feel free to contribute in these discussions, or raise an issue through Github itself.

**Reply by aimylee** [reply]
Looks like there are reactions too. Interesting. Is this something new?

**Reply by chanjuping** [reply]
I noticed the emoji reactions on Github a few years back, so maybe not really new, but not too old either. I personally enjoy them immensely!
